### ISSUE TITLE
new test doesn't appear in Descartes report 

### ISSUE THREAD

**Issue by assadOW2** [STAMP-project/pitest-descartes#85]
#### Characteristics
- **Issue Type**: [test report]
- **Execution Environment**: [ubuntu 16.04]
- **Reporter**: [Assad Montasser, am@ow2.org]

#### Description

During this commit: https://gitlab.ow2.org/sat4j/sat4j/commit/6c8c8c13d2743462d070c322bb10bffcf0b67374
a new Test class was added to the original suite in order to detect mutation of org.sat4.minisat.core.Heap.increase() 

The execution of Descartes doesn't detect the new test in the report: https://github.com/STAMP-project/descartes-usecases-output/tree/master/OW2/Sat4j/201811120812

And the mutant is still not killed. 

#### [Help on issue template](https://github.com/STAMP-project/pitest-descartes/blob/master/.github/ISSUE_DOC.md) 
Preview to follow the link or open file .github/ISSUE_DOC.md

**Comment by oscarlvp** [comment]
Could you please include the configuration used to execute Descartes?

**Comment by assadOW2** [comment]
Sure, this is the one you gave me last time in Paris.
Here it is:

```
<profile>
		<id>descartes</id>
		<build>
			<plugins>
				<plugin>
					<groupId>org.pitest</groupId>
					<artifactId>pitest-maven</artifactId>
					<version>1.4.0</version>
					<configuration>
						<verbose>true</verbose>
						<mutationEngine>descartes</mutationEngine>
						<outputFormats>
							<value>HTML</value>
							<value>CSV</value>
							<value>ISSUES</value>
							<value>METHODS</value>
						</outputFormats>
	  				        <jvmArgs>
							<value>-ea</value>
							<value>-Dtest.prefix=/home/assadm/OW2-Projects/Forks/sat4j/org.sat4j.core/src/test/testfiles/</value>
							<value>-Dtest.pbprefix=/home/assadm/OW2-Projects/Forks/sat4j/org.sat4j.core/src/test/testfiles/</value>
						</jvmArgs>
						<excludedTestClasses>
							<!--
							<exclude>**/AllTests*</exclude>
							<exclude>**/XplainTests*</exclude>
							<exclude>**/GoodOPBReaderTest*</exclude>
							<exclude>**/Abstract*Test*</exclude>
											<exclude>**/CompetPBResMinHTMixedConstraintsObjectiveLearningTest*</exclude>
								<exclude>**/CompetPBResHTMixedConstraintsObjectiveLearningTest*</exclude>
							<exclude>**/MiniOPBLongClauseCardMinLearningTest*</exclude>
							<exclude>**/PBCPMixedConstraintsCautiousTest*</exclude>
							<exclude>**/MultiObjectiveDependencyHelperTest*</exclude>
							<exclude>org.sat4j.pb.CardConstrFinderTest.class</exclude>
						-->
							<param>org.sat4j.minisat.M2MiniLearningTest</param>
							<param>org.sat4j.minisat.M2AtLeastTest</param>
									
						</excludedTestClasses>
					</configuration>
				        <dependencies>
				                  <dependency>
							<groupId>eu.stamp-project</groupId>
							<artifactId>descartes</artifactId>
							<version>1.2.4</version>
						</dependency>
					</dependencies>
				</plugin>
			</plugins>
		</build>
	</profile>
```
When I execute Descartes I use this command line:

mvn org.pitest:pitest-maven:mutationCoverage  -Pdescartes

**Comment by oscarlvp** [comment]
The commit included in the comment shows the modification of the existing test class `HeapTest`. What seems to be new is the code of the `testIncrease` method, that indeed is testing `Heap.increase` and it is not included in the report as @assadOW2 says.
Now, I cloned the SAT4J repository in its current revision (958ad97a) I built the code and executed PIT with this configuration and the report does include the test and shows that the mutant has been killed and there is no issue mentioning the `increase` method.
 
![screenshot 2018-11-15 at 11 49 32](https://user-images.githubusercontent.com/630717/48548278-d9367f80-e8cc-11e8-8258-b02c8cce1089.png)
